components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/internal/IdentitySAMLSSOServiceComponent.java

```diff
@@ -109,10 +109,10 @@ protected void activate(ComponentContext ctxt) {
 
 
             String redirectHtmlPath = CarbonUtils.getCarbonHome() + File.separator + "repository"
-                    + File.separator + "resources" + File.separator + "security" + File.separator + "sso_response.html";
+                    + File.separator + "resources" + File.separator + "identity" + File.separator + "pages" + File.separator + "samlsso_response.html";
```

Comment:
Why samlsso_response.html is not inside the "sso_responses" folder, while the samlsso_federate.html is.